### ISSUE TITLE
How will this site work with social media channels?

### ISSUE THREAD

**Issue by vegantriathlete** [vegantriathlete/how-to-brand-your-website#4]
Can the channels be used as a means for interactivity?

**Comment by vegantriathlete** [comment]
Yes, let's use Facebook page / group, Pinterest and Twitter. The site should be optimized to take advantage of those channels. Let's make sure that it's easy to share stuff from the site.

**Comment by kpalana** [comment]
Absolutely. I personally don't use Instragram, but it might be worth considering if you plan to have lots of recipes and therefore images to post to it.

**Comment by vegantriathlete** [comment]
Interesting thought about photos being used in recipes. Of course it's important to have photos with the recipes. I hadn't even considered that!

So do you think Pinterest or Instagram would be better? I am a little nervous about taking on more than three social media platforms and how much time it will take to keep all of them active.

Out of Facebook, Twitter, Pinterest and Instagram which three would you pick?

**Comment by kpalana** [comment]
I would choose Facebook and Twitter for sure. Since Pinterest has a more female user base, I might choose that one since I suspect you will get mostly women (Moms) at your site.
